terraform/environments/xhibit-portal/prtg-load-balancer.tf

```diff
@@ -0,0 +1,537 @@
+# data "aws_ec2_managed_prefix_list" "cf" {
+#   name = "com.amazonaws.global.cloudfront.origin-facing"
+# }
+
+# to get account id
+# data "aws_caller_identity" "current" {}
+
+# resource "aws_security_group_rule" "allow_cloudfront_ips" {
+#   depends_on        = [aws_security_group.prtg_lb]
+#   security_group_id = aws_security_group.prtg_lb.id
+#   type              = "ingress"
+#   description       = "allow web traffic to get to ingestion server"
+#   from_port         = 443
+#   to_port           = 443
+#   protocol          = "TCP"
+#   prefix_list_ids   = [data.aws_ec2_managed_prefix_list.cf.id]
+
+# }
+
+data "aws_subnets" "prtg-shared-public" {
+  filter {
+    name   = "vpc-id"
+    values = [local.vpc_id]
+  }
+  tags = {
+    "Name" = "${var.networking[0].business-unit}-${local.environment}-${var.networking[0].set}-public*"
+  }
+}
+
+resource "aws_lb" "prtg_lb" {
+
+  depends_on = [
+    aws_security_group.prtg_lb,
+  ]
+
+  name                       = "prtg-lb-${var.networking[0].application}"
+  internal                   = false
+  load_balancer_type         = "application"
+  security_groups            = [aws_security_group.prtg_lb.id]
+  subnets                    = data.aws_subnets.prtg-shared-public.ids
+  enable_deletion_protection = false
+
+  access_logs {
+    bucket  = aws_s3_bucket.loadbalancer_logs.bucket
+    prefix  = "http-lb"
+    enabled = true
+  }
+
+  tags = merge(
+    local.tags,
+    {
+      Name = "prtg-lb-${var.networking[0].application}"
+    },
+  )
+}
+
+resource "aws_lb_target_group" "prtg_lb_web_tg" {
+  depends_on           = [aws_lb.prtg_lb]
+  name                 = "prtg-lb-web-tg-${var.networking[0].application}"
+  port                 = 80
+  protocol             = "HTTP"
+  deregistration_delay = "30"
+  vpc_id               = local.vpc_id
+
+  health_check {
+    path                = "/Secure/Default.aspx"
```

Comment:
This health check won't work for the PRTG server

Reply:
Health check removed;
resource "aws_lb_target_group" "prtg_lb_web_tg" {
  depends_on           = [aws_lb.prtg_lb]
  name                 = "prtg-lb-web-tg-${var.networking[0].application}"
  port                 = 80
  protocol             = "HTTP"
  deregistration_delay = "30"
  vpc_id               = local.vpc_id

#   health_check {
#     path                = "/Secure/Default.aspx"
#     port                = 80
#     healthy_threshold   = 6
#     unhealthy_threshold = 2
#     timeout             = 2
#     interval            = 5
#     matcher             = "302" # change this to 200 when the database comes up
#   }

  tags = merge(
    local.tags,
    {
      Name = "prtg-lb_-g-${var.networking[0].application}"
    },
  )
}

Reply:
Bold is now commented in code

Reply:
You still need a health check for the instance - so making sure the login page is available for example.

Reply:
THanks Ben, using path = "/index.htm" now